server/queries/findAdminById.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
```

Comment:
don't commit empty tests please :)

Reply:
Fixed in `jwt-authentication-login-page` branch